Loop/Managers/LoopDataManager.swift

```diff
@@ -199,6 +199,7 @@ final class LoopDataManager: ObservableObject {
             ) { (note) in
                 Task { @MainActor in
                     self.logger.default("Received notification of glucose samples changing")
+                    self.restartGlucoseValueStalenessTimer()
```

Comment:
Should also start the timer on app start, I think.  I didn't see that happening, but maybe I missed it?